LocaleUtils/index.js

```diff
@@ -0,0 +1,21 @@
+
+export async function assertNoEnglishLeaks(mockFile, viewText, noTranslationContent) {
+  const regEx = '(?<=\》).+?(?=\《)'; // eslint-disable-line no-useless-escape
+  const pseudoLocSymbols = ['》', '《', '《 》', '《》'];
+
+  // Exclude all elements with a class of no-translate. Example phone numbers
+  if (noTranslationContent.length) {
+    viewText = viewText.replace(noTranslationContent, '');
+  }
+
+  let extractedString = viewText.trim().replace(new RegExp(regEx, 'g'), ' ');
+  extractedString = extractedString.split(' ');
+  const enLeaks = extractedString.filter( (item) => {
+    return item.length && !pseudoLocSymbols.includes(item.trim());
+  });
+
+  if (enLeaks.length) {
+    const error = enLeaks.join(' ');
+    throw new Error(`Hard coded english text found for mock ${mockFile}: ${error}`);
```

Comment:
Instead can we say `This string is not localized ${mockFile}: ${error}`

Reply:
Going to update to `Unlocalized text found in mock ${mockFile}: ${error}`